node/src/BaseClient.ts

```diff
@@ -5538,10 +5541,12 @@ export class BaseClient {
      * ```
      */
     public async pfmerge(
-        destination: string,
-        sourceKeys: string[],
+        destination: GlideString,
+        sourceKeys: GlideString[],
     ): Promise<"OK"> {
-        return this.createWritePromise(createPfMerge(destination, sourceKeys));
+        return this.createWritePromise(createPfMerge(destination, sourceKeys), {
+            decoder: Decoder.String,
```

Comment:
For return `OK`, do we need to specify `decoder: Decoder.String`?

Reply:
I think yes

Reply:
However, in this PR, I see this comment:
https://github.com/valkey-io/valkey-glide/pull/2172/files#r1724882473